### PR TITLE
Support Joining a Game

### PR DESCRIPTION
**Pull request by snsmith92**
Feedback on this code guys? Task: 

> Add a link to the page that lists the open seat games to join the game. Have the action trigger the games#update method and change the current game’s black_player_id to the currently logged in user id.

Also, I was unsure whether to use `put` or `patch` for the rspec tests so what do you think?

### CODE REVIEW

**Comment by LampshadeWarrior** [comment]
Hey Sherzel,

It looks really good to me! There's just a couple small issues.

1) From checking Codeship, it looks like you've got an extra `end` somewhere in Piece.rb, maybe around line 33?

screen shot 2018-02-13 at 8 58 40 pm

2) It also looks like there's no view for the index action, which is making rspec spit out an error.

Other than that, I think it's on point!

File: spec/controllers/games_controller_spec.rb [hunk 1]
```diff
@@ -13,5 +13,22 @@
       get :index
       expect(response).to have_http_status(:success)
     end
+  end
+
+  describe "games#update action" do
```
Comment (ilyakrasnov) [inline spec/controllers/games_controller_spec.rb]: Nice tests 👍

**Comment by ilyakrasnov** [comment]
Other than the comments above, looks good 👍

**Comment by snsmith92** [comment]
Thanks @ilya and @LampshadeWarrior ! I removed the trailing 'end'. However there is a index view so not sure why that's a problem?

**Pull request merged**